### ISSUE TITLE
Support for Light typeface

### ISSUE THREAD

**Issue by pakLebah** [SSNikolaevich/DejaVuSansCode#13]
I like this font. However, it's a bit too thick to my taste. Could you provide a Light typeface variant? Just like in Fira Code font. Thank you. 😊

**Comment by SSNikolaevich** [comment]
I just added a few glyphs to DejaVu Sans Mono. There is no light variant of this font in DejaVu Fonts repository.

**Comment by pakLebah** [comment]
Yes, I knew there's no Light variant in DejaVu Sans Mono. But I thought you could possibly add it. Well, if it's not possible, it's alright then. 

Perhaps you could add more ligatures so it can be as rich as Fira Code's. I like this font more than Fira Code, but the ligatures and Light variant offered by Fira Code can't be ignored. 😊
